x-pack/plugins/security/public/management/role_mappings/edit_role_mapping/rule_editor_panel/json_rule_editor.tsx

```diff
@@ -76,25 +77,26 @@ export const JSONRuleEditor = (props: Props) => {
       data-test-subj="roleMappingsJSONEditor"
     >
       <Fragment>
-        <EuiCodeEditor
+        <CodeEditor
           aria-label={''}
-          mode={'json'}
-          theme="github"
+          languageId={XJsonLang.ID}
           value={rawRules}
           onChange={onRulesChange}
-          width="100%"
-          height="auto"
-          minLines={6}
-          maxLines={30}
-          isReadOnly={false}
-          setOptions={{
-            showLineNumbers: true,
+          fullWidth={true}
+          height="300px"
+          options={{
+            accessibilitySupport: 'off',
```

Comment:
Disabling `accessibilitySupport` sounds bad, but I think it's necessary. This prevents Monaco from enabling shortcuts such as these that appear to be selected by default on macOS:

![image](https://user-images.githubusercontent.com/5295965/127913496-596cebf2-d559-43f7-9b7e-806d6aa8f2a1.png)

None of these should be used when editing code. It just causes frustration when typing.